src/components/PostInfo/PostInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { PreparedPost } from '../../typedefs/PreparedPost';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import '../PostList/PostList.scss';
+
+interface Props {
+  singlePost: Omit<PreparedPost, 'userId' | 'id'>
+}
+
+export const PostInfo: React.FC<Props> = ({ singlePost }) => {
+  const {
+    title,
+    body,
+    user,
+    comments,
+  } = singlePost;
+
+  return (
+    <li className="post-list__list-item">
+      <h2 className="post-list__list-item-title">
+        {title}
+      </h2>
+      <p className="post-list__list-item-text">
+        {body}
+      </p>
+
+      <UserInfo userInfo={user} />
```

Comment:
Your user prop might be undefined so use conditional rendering here, `&&` operator will do it
```suggestion
      {user && <UserInfo userInfo={user} />}
```